tests/php/EnqueueScriptTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Distributor;
+
+use WP_Mock\Tools\TestCase;
+
+/**
+ * @since x.x.x
+ */
+class EnqueueScriptTest extends TestCase {
```

Comment:
Do you need any assistance with the tests? the mocks can make them a little complex.

Reply:
@peterwilsoncc I added to update them later when I implement run tests on WordPress installation instead of mocks.